programs/blktests/meta.yaml

```diff
@@ -0,0 +1,27 @@
+description: >
+ a test framework for the Linux kernel block layer and storage stack.
+ It is inspired by the xfstests filesystem testing framework. 
+ It was originally written by Omar Sandoval and announced in 2017.
+
+homepage: https://github.com/osandov/blktests
+
+parameters:
+ test:
+ nvme_trtype:
+ use_siw:
+
+results:
```

Comment:
no need to have results for this functional test

Reply:
ok

Reply:
Is this resolved? The results are still there

Reply:
ok